Address feedback from internal test (#4)
USEPA/supply-chain-factors: README.Rmd

3. A local version of this repository, preferably a [release](https://github.com/USEPA/supply-chain-factors/releases) version to create EPA-verified factors.

### Optional Requirements
1. [Git](https://github.com/git-guides/install-git), if users wish to clone the code from this repository using Git, instead of downloading a zip file of the code.
2. [RStudio](https://www.rstudio.com/products/rstudio/download/#download), which provides a graphical interface and button to run this code (see Usage).

## Usage

The code in this package can be used to generate the USEEIO-based Supply Chain GHG Emission Factors, based on options selected by the user at run-time. 

If users choose to download a zip file of the code, they can unzip the file then [create a project from the existing directory within RStudio](https://support.rstudio.com/hc/en-us/articles/200526207-Using-RStudio-Projects) by selecting the unzipped local directory of files.

> If users choose to clone this repository, they can [create a project by cloing the repository with RStudio](https://resources.github.com/whitepapers/github-and-rstudio/#:~:text=Clone%20the%20repository%20with%20RStudio).

Reviewer: I think its should rather be

"If users have git installed, the users can create a project by cloning the repository with RStudio"

Author: ```suggestion
If users have git installed, they can [create a project by cloning the repository with RStudio](https://resources.github.com/whitepapers/github-and-rstudio/#:~:text=Clone%20the%20repository%20with%20RStudio).
```

> Once a project is created, double click to open it, then open the *CalculateEmissionFactors.Rmd* in the editor, click on *Knit* and select *Knit with Parameters…*.

Reviewer: I don't have to open the project by double-clicking ... once its created its automatically opened

Author: ```suggestion
Once a project is created and opened, then open the *CalculateEmissionFactors.Rmd* in the editor, click on *Knit* and select *Knit with Parameters…*.
```

Author: Under the 'clone' option, users have to double click to open it.
